docker-compose-github.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  tests:
+    platform: linux/amd64
+    build: .
+    environment:
+      DEBUG: "True"
+      PYTHONPATH: /opt/app
+      TEST_SYSTEM_HOST:
```

Comment:
No, I think you missed how docker compose works. If you define it like this, it takes them from the system environment, which is provided by github actions

Reply:
Okay, I did think about that, but wasn't sure since these variables were also specifically declared in `.github/workflows/test.yml`, where it directly links to the secrets file. But I don't know how github actions work, hence the confusion 😉